app/lib/ga-log-adapter.js

```diff
@@ -0,0 +1,48 @@
+const dimensionMap = {
+  experiment: 1,
+  cohort: 2,
+  projectToken: 3,
+  userID: 4,
+  subjectID: 5,
+  workflowID: 6
+};
+
+class GALogAdapter {
+  constructor(gaLayer) {
+    this.gaLayer = gaLayer;
+  }
+
+  configure(keys) {
+    this.onRemember(keys);
+  }
+
+  onRemember(newKeys) {
+    Object.keys(newKeys).forEach((key) => {
+      if (Object.keys(dimensionMap).includes(key)) {
+        this.gaLayer('set', `dimension${dimensionMap[key]}`, newKeys[key]);
```

Comment:
It looks like this is firing when the page is loaded, and I'm getting `this.gaLayer is not a function`

Reply:
oh that makes sense; that will happen in places where the google analytics tracking code hasn't been installed. on production it should work fine though

Reply:
The error won't let the page load.